README.md

```diff
@@ -410,6 +410,27 @@ For example this means that by default we upgrade the minimum version of Airflow
 to 2.3.0 in the first Provider's release after 11th of October 2022 (11th of October 2021 is the date when the
 first `PATCHLEVEL` of 2.2 (2.2.0) has been released.
 
+Since providers are often connected with some stakeholders that are vitally interested in maintaining
+their integrations (for example cloud providers, ore specific service providers), and we are also bound with
+the Apache Software Foundation release policies. The provider's governance model is something we name
+"mixed governance".
+
+The Airflow Community and release manager still decides when to release those providers.
+This is fully managed by the community and the usual release-management process.
```

Comment:
IMO this paragraph doesn't hold any useful information. As a reader, in case I would have a question; what are useful resources or channels to ask?

Reply:
This is (not very precisely explained but I just changed it) the "governance model". And I've added the link to the ASF relese policies that not only explain the mechanics, reasoning but also  have a bunch of FAQs to answer.